### PR TITLE
Stunbatons can now only be used during Red Alerts

### PR DESCRIPTION
**Pull request by optimumtact**
## About The Pull Request
This prevents stunbatons being turned on to full stun mode if the station is not in a red alert emergency.

## Why It's Good For The Game

The decision to limit the use of stunbatons in TGStation to red alert emergencies is due to the general degradation of the melee combat design space in the game. The stunbaton's ability to stun and knock down opponents with ease has made it the go-to weapon for many players, leading to a lack of diversity in melee combat strategies.

This over-reliance on stunbatons has made other melee weapons, such as knives or clubs, obsolete in many situations. This not only limits player choice but also reduces the overall challenge and fun of melee combat in the game.

By restricting the use of stunbatons to red alert emergencies, players are encouraged to explore other melee weapons and develop new strategies, making the game more engaging and challenging. It also ensures that the use of stunbatons is reserved for critical situations, making combat encounters more meaningful and immersive.

## Changelog

:cl: oranges
add: Stunbatons now require a red alert to activate stun mode
/:cl:

### CODE REVIEW

File: code/game/objects/items/melee/baton.dm
Comment (san7890) [inline code/game/objects/items/melee/baton.dm:526]:
```suggestion
		else
```

?

Comment (tralezab) [reply code/game/objects/items/melee/baton.dm:526]: pythonbros...

Comment (san7890) [reply code/game/objects/items/melee/baton.dm]: YO UCAN'T FORCE PUSH OVER YOUR CHANGES WE ALL SAW THIS

File: code/game/objects/items/melee/baton.dm
Comment (san7890) [inline code/game/objects/items/melee/baton.dm:527]:
```suggestion
			balloon_alert(user, "This is not an emergency situation! You find yourself cripplingly unable to use your stun baton, since you have a steadfast and righteous respect for the rules and regulations of this station!")
```

reads better imo

Comment (optimumtact) [reply code/game/objects/items/melee/baton.dm:527]: please stop making troll suggestions

Comment (optimumtact) [reply code/game/objects/items/melee/baton.dm:527]: and balloon alerts are not recommended for self interactions

**Comment by NamelessFairy** [comment]
Two things, is this intended to also impact cattleprods, just want to make sure as it currently does.
Secondly, the way this is coded means that people can turn on the baton during a red alert and keep it on when the red alert ends, additionally they cannot turn it off when the red alert ends which I assume is an oversight?

**Comment by optimumtact** [comment]
oops, I forgot to allow disabling it

File: code/game/objects/items/melee/baton.dm
Comment (Iamgoofball) [inline code/game/objects/items/melee/baton.dm:522]: We have a signal for security level changes, you can tap into that to automatically shut batons off when it drops from red.

However, what's stopping crew from roundstart keeping it on red all day?

**Review by Iamgoofball — APPROVED** [review]

File: code/game/objects/items/melee/baton.dm
Comment (optimumtact) [reply code/game/objects/items/melee/baton.dm:522]: RP

**Comment by ZephyrTFA** [comment]
Damn this is based, getting people to use more than a single weapon.
Although this will just make people default to the stunprod probably

**Comment by Zytolg** [comment]
Anything that forces red alerts to be used more is beneficial. Thumbs up from me. We should also open the floor up to traitoring opportunities here such as using a cmag to bypass the red-alert restrictions.

File: code/game/objects/items/melee/baton.dm
Comment (NamelessFairy) [inline code/game/objects/items/melee/baton.dm:528]:
```suggestion
			if(!active)
```

**Comment by Paxilmaniac** [comment]
Weak solution, remove them

**Comment by Sightld2** [comment]
Does this affect the detective's police baton? It'd feel weird if it did, but at the same time now the detective is the only one with a baton at roundstart.

File: code/game/objects/items/melee/baton.dm
Comment (NamelessFairy) [reply code/game/objects/items/melee/baton.dm:528]: Oh actually this still wont work, if the stations on red alert since this is behind an else its not going to reach this code

**Comment by NamelessFairy** [comment]
> Does this affect the detective's police baton? It'd feel weird if it did, but at the same time now the detective is the only one with a baton at roundstart.

It does not

**Comment by TheRealWoodMan** [comment]
This is really cool, one change you could make if you still wanna let sec have access to them in certain situations is letting them still work on people who are tagged as Wanted on the brig console. That would help encourage Sec to use the consoles more instead of only relying on radio, and you could also implement traitor items into all of this as suggested above. Hope this gets passed.

**Comment by LemonInTheDark** [comment]
we are entering the darkest possible timeline lads, buckle up

File: code/game/objects/items/melee/baton.dm
Comment (Iamgoofball) [reply code/game/objects/items/melee/baton.dm:522]: you expect the admin team to enforce that? they barely stop AIs from cracking the upload open for the HOS when nuke ops are announced, RP is not a priority for our admins to enforce

**Comment by UnokiAs** [comment]
As I walked in a corridor, I saw a orange blur at the end of it... with a big smile showing teeth suddently a creepy song played, I knew this song very well... It was Eduard Khil going trololo...
It was already to late to do anything...

File: code/game/objects/items/melee/baton.dm
Comment (san7890) [inline code/game/objects/items/melee/baton.dm:75]: This doesn't actually de-activate it?

**Comment by Korusho** [comment]
Then how are security supposed to reliably detain basic criminals and shitters reliably without becoming glorified loot pinatas and/or having to brutalise people into compliance? This is a fire extinguisher to a burning match. If you have issues with stuns, you are using going the laziest possible route. Introduce stun immunity to armors that antags use/can acquire, add partial stun resistance to low tier armor, but not this. 

However, as I have stated, this is the most brute force way to deal with stuns. It is not, however, the most intelligent.

File: code/game/objects/items/melee/baton.dm
Comment (LemonInTheDark) [inline code/game/objects/items/melee/baton.dm:541]: This proc is desperately crying out for early returns
We should have a setter for active that checks if the cell has charge/exists, and only plays the sound if it does

Would make this significantly less stupid

**Comment by SethLafuente** [comment]
be more reasonable and change it to blue alert will ya

File: code/game/objects/items/melee/baton.dm
Comment (LemonInTheDark) [inline code/game/objects/items/melee/baton.dm:75]: Does this mean I could toggle to red alert, turn on all my batons, and then flip it back and have the only active batons on the station?
Also why is this off /baton and not /baton/security? do you intend this to apply to telescoping batons?

**Comment by improvedname** [comment]
terry red alert roundstart meta begins here...
make it delta alert tbh

**Comment by tralezab** [comment]
i would componentize the code 😎

**Comment by LemonInTheDark** [comment]
I would kill armhulen

File: code/game/objects/items/melee/baton.dm
Comment (ZephyrTFA) [inline code/game/objects/items/melee/baton.dm:378]: extra set of && here?

**Comment by newfren** [comment]
makes the traditional equipment crate a lot better

**Comment by optimumtact** [comment]
>Then how are security supposed to reliably detain basic criminals and shitters reliably without becoming glorified loot pinatas and/or having to brutalise people into compliance? 

> having to brutalise people into compliance?

I don't see the problem?

**Comment by tralezab** [comment]
> > Then how are security supposed to reliably detain basic criminals and shitters reliably without becoming glorified loot pinatas and/or having to brutalise people into compliance?
> 
> > having to brutalise people into compliance?
> 
> I don't see the problem?

The brig physician job would go along with this well

**Comment by improvedname** [comment]
As if arresting a greyshirt wasn't hard enough with a baton i wonder how this wl stray maybe have the baton itself do like a minor amount of stamina damage when turned off like 10 stamina damage with 5 brute

**Comment by dendyd00m** [comment]
wordlessly batonning people is an issue but this is a lazy solution. best outcome is hos will be loudvoicing for red alert on the command channel 8 nanoseconds into every round and worst outcome is sec will be beating people into crit to be able to handcuff them. the reason there is a lack of diversity in the melee combat system isn't because of stun batons it's because of the melee combat system. 

I agree with the other poster who mentioned having them only apply stuns to people who are wanted. that means sec actually have to have a conversation and rub the sec department's shared 2 brain cells together to figure out who is guilty of a crime before they mobilize on them. this is better than sending the entire station into lockdown every time someone needs to be put in cuffs.

**Comment by tralezab** [comment]
> best outcome is hos will be loudvoicing for red alert on the command channel 8 nanoseconds into every round

thats an admin problem

> worst outcome is sec will be beating people into crit to be able to handcuff them

just don't resist

> the rest

doesn't actually bring diversity into the weapon pool or really solve the issue

**Comment by optimumtact** [comment]
how would a baton know who's wanted, it's not an AI

**Comment by Zytolg** [comment]
> how would a baton know who's wanted, it's not an AI

But if we gave the AI a stunbaton...

**Comment by Cheshify** [comment]
Interesting PR honestly. If the goal is to limit stun combat and open the floor to more forms of combat, is this a sign of a future redesign to melee as a whole? I want to see where this ends up going!

**Comment by Kubisopplay** [comment]
The wanted thing would be hilarious honestly, since I can imagine the "AI SET EVERYONE TO ARREST" terry meta.

**Comment by Rex9001** [comment]
I assume this pr also signals a shift in your concerns for solo antagonists and how reliant they are stun batons?

**Comment by Vect0r2** [comment]
I really don't like this... I do not play sec, but this leaves a icky taste in my mouth

**Comment by wesoda25** [comment]
I mean this isn't great but it's better than what we've got.

**Comment by SethLafuente** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=33&t=33438&p=666352#top

there probably should be a talk if we are simply allowed to render this PR useless in LRP or at least an admin decision supporting this PR if it's actually serious because this is going very much into admin and IC territory for a code addition without any before talk

File: code/game/objects/items/melee/baton.dm
Comment (dragomagol) [inline code/game/objects/items/melee/baton.dm:78]:
```suggestion
		to_chat(user, span_alert("Your baton turns off as the threat to the station has passed."))
```

File: code/game/objects/items/melee/baton.dm
Comment (dragomagol) [inline code/game/objects/items/melee/baton.dm:379]:
```suggestion
		to_chat(user, span_warning("You cannot use your baton during this emergency level."))
```

File: code/game/objects/items/melee/baton.dm
Comment (dragomagol) [inline code/game/objects/items/melee/baton.dm:555]:
```suggestion
				to_chat(user, span_warning("You cannot use your baton in a non-emergency situation."))
```

**Review by dragomagol — COMMENTED** [review]
Small grammar stuff

**Comment by Jolly-66** [comment]
Honestly quite incredible.

**Comment by Bluedino1025** [comment]
this is a horrible idea for a whole list of reasons.

stun combat is an issue but it’s because of the melee system, who ever is on the ground first will always lose. so if the baton is gone there’ll be something else officers will use thats just as bad.
speaking of which, the baton conversion kit, this will just get used every shift so we can actually do our jobs.

the flash and pepperspray don’t stun long enough to cuff. the bola will only slow down. the disabler doesn’t work in crowds.

what this will inspire is just dropping the suspect to crit or using makeshift stuns. it’s a coder solution to a player problem. 
what could work better is the idea of only working on wanted people or reducing the stamina damage it does, make it need three or four hits to drop someone instead of two.

**Comment by tralezab** [comment]
> I mean this isn't great or really even good but I think any attempt at shifting us away from the batong meta is at least worth a try.

yeah that's what i'm thinking as well

**Comment by dendyd00m** [comment]
> thats an admin problem

you can call it that all you like but if it's a code fix to a behavioural problem which doesn't solve the behavioural problem then it's not a good fix. gating batons behind red alert and then asking admins to punish people who red alert incorrectly under policy 4 subsection 7 subclause B is scuffed.

**Comment by UnokiAs** [comment]
Can this be speed merge please !

**Comment by itseasytosee** [comment]
None of you understand the true brilliance of this PR. The amount of scientific data that we will gain from this ill be revealing and profound. Also we don't have any clubs in this game that I know of.

**Comment by Petnavis** [comment]
Is this going to address the batons that instant drop people (ordered from cargo or round start from det/heads) that allow for people immediately shoving them which forces them into a stunlock? Because that's what's going to happen. Instead of people using the stun batons sec is just going to order the classic ones and stam stun people and then beat them into crit. It doesn't address the issue whatsoever and doesn't address the braindead baton gameplay.

How many rounds do we need of some guy running around on god knows what from Chem baton -> shoving people with a fireaxe?

**Comment by the-orange-cow** [comment]
I realize that this will probably be considered an admin issue, but a result of this will likely be a major increase sec prematurely critting anyone they feel the need to question.

**Comment by steroidberg** [comment]
My god! This man is truly based!

**Comment by nickup9** [comment]
**BLUF: This is a massive balance change and needs parallel changes alongside it to make it work. I advise against merging this PR until those changes are ready to be presented with this PR. I play most of the time but I can try to lend a hand with these changes if necessary.**

> I don't see the problem?/Just don't resist/just harmbaton then  

There's issues with the AI for one, as an asimov AI would take issue with Sec harm batoning humans if sec had no other means to defend themselves or arrest a target. An example might be a traitor with a esword or double e-sword who is partially or completely immune to disablers and ranged laser fire in general. And if a harmbaton is intended to arrest a target as mentioned, everyone knows that the moment a harmbaton is used to get compliance from a suspect, the suspect can cry law 1 if human and Sec will have one of the most powerful entities on the station working against them from that moment on. 

Overall people are more likely to cry shitsec or police brutality. It seems based off Trazelab and Orange's comments that harmbatoning is intended as a threat against suspects running away, but since this is a video game it's likely that people will decide to resist and/or cry shitsec or law 1. The remedy to this could be to force compliance by admin ruling, but for our LRP servers I advise against this becoming necessary. Maybe Asimov could be rewritten to ignore harm up to a certain point, but that doesn't seem proper as well.

The above has already been mentioned, but I'm trying to give examples to make it more clear and paint a picture of how these changes might play out.


**Other considerations**
The following don't necessarily mean this PR should not be removed, but the issues presented are targets for rebalance, overall it'd be wise to consider the following:

_Nukies:_
As a result of this PR nukies could also roll in without a wardec and with energy shields, making them immune to ranged fire from the crew. In this case quickest remedy could be to make use of the riot shotgun and its beanbag rounds - but it's my understanding that this codebase wants to move past ballistics for the crew. Also Asimov comes into play again here, not as relevant for nukies but more so for the traitor example.

_Changeling:_ 
Massive buff to adrenals I believe, as a baton at least knocks down the changeling. A disabler will do practically nothing since it causes no knockdown on hit, and getting a stamkill on the target requires a user get their shots on the changeling moving at full speed even after it's taken several hits. This is before we get into the matter of the changeling's EMP draining disabler charge.

_Revolutionary:_
Main advantage revs have are numbers and stealth to a point. Main advantage security has is better gear...except now that their primary self defense tool is gone, security can easily get mobbed and taken out before the red alert is set. Flashbangs and pepper spray are useful of course, but both are only good for stunning and slowing crowds for a relatively short amount of time, and pepper spray especially is easy to evade in open areas. A disabler against a crowd will fail in most cases.

_Cultist:_
See above, except that the gear gap is smaller to begin with.

**Counterpoints to concerns**
_Just red alert_
The simple solution to these issues would be to simply set red alert when the threat is spotted. This isn't a major issue to be fair. However, current gameplay requires the use of a baton for a security member to buy time to report the threat to others, assuming that they were jumped by the threat. Without a baton it's likely that threat can go unnoticed for much longer as they can move more aggressively to silence the reporter without fear of getting stunned. Also, see the nukies example above.

_Use other tools (e.g. disablers, flashbangs, pepper spray)_
I agree that other tools should be made use of more. This PR is a good step in that direction, however without proper balancing these tools won't work efficiently enough or fast enough to be of use. A disabler requires around three shots to even slow a target, 4-5 to stun them. A flashbang works very well usually, but is one time use and may falter without a stun baton to follow up with. Pepper spray is slow and security masks are not immune to it.

_Nanotrasen have lore reasons to limit baton usage because of public relations/costs savings/energy savings concerns_
I can get behind this and don't have a counter to it, it's on brand for NT.

_The crew can lend a hand and provide numbers_
This is true on LRP servers, however on MRP due to the rule against valid hunting this is not as true. I understand that this is a server rules issue and not a codebase issue, but even so I think it bears mentioning.

**Overall:**
As it stands this change will heavily unbalance crew/sec and antag metas. I advise holding off on merging this and keeping this as a test merge at most until other changes are introduced in parallel.

**Comment by Acer202** [comment]
Honestly? Anything to get rid of stun combat. Nuke it from orbit.

**Comment by YuliceTheDruid** [comment]
This smells like a balance/feature PR, aren't we in a feature freeze?

**Comment by nickup9** [comment]
> This smells like a balance/feature PR, aren't we in a feature freeze?

freeze exempt

**Comment by YuliceTheDruid** [comment]
On a less shitposty note can we perhaps give sec a more robust fighting tool since every other form of non stun based weaponry was kneecapped before this, lest we just get legions of det baton crafting in the mean time?

**Comment by san7890** [comment]
> On a less shitposty note can we perhaps give sec a more robust fighting tool since every other form of non stun based weaponry was kneecapped before this, lest we just get legions of det baton crafting in the mean time?

can't you order it from cargo?

**Comment by Sightld2** [comment]
> > On a less shitposty note can we perhaps give sec a more robust fighting tool since every other form of non stun based weaponry was kneecapped before this, lest we just get legions of det baton crafting in the mean time?
> 
> can't you order it from cargo?

You order the kit and craft it I believe

**Comment by DaydreamIQ** [comment]
So I'm all for nerfing stun batons but ain't this just gonna result in more people going red alert the second anything starts happening?

**Comment by improvedname** [comment]
just dualwield dragnets pointblank deals 120 stam damage lol

**Comment by Timberpoes** [comment]
I just want to say that I am here, now, in this timeline. I just want future generations to know I was here.

**Comment by LSommers** [comment]
Players will tell you they hate sec beating the fuck out of them.
Coders will remove the last reliable stun sec has.

Brilliant.

**Comment by LemonInTheDark** [comment]
@optimumtact you happy with the changes here? s hard to tell if all the reviews have been addressed with the forcepushes

**Comment by LemonInTheDark** [comment]
231nwdsa9-2wendsnw
I love living in the darkest timeline

**Comment by castawaynont** [comment]
lol

**Comment by ArcaneMusic** [comment]
Not too late to be in the screenshot

**Comment by drIcepop** [comment]
honestly quite incredible

**Comment by Contrabang** [comment]
Include me in the screenshot

**Comment by BeeSting12** [comment]
I lol'd when I read the post about crew helping on LRP. Maybe against an antagonist, but against greytide nothing will happen. May as well remove security as this point tbh

**Comment by IndieanaJones** [comment]
Fun

**Comment by Mitchs98** [comment]
I have no words other than well played.

**Comment by dendyd00m** [comment]
let future generations know i did my best to push back against the extinction event

**Comment by OverwatchVoice** [comment]
Watch this get rolled back in a few days.
Seriosly, I have nothing against nerfing the stun batons. They're an instant win button in most cases already, but this..?
You could've made it a three-hit or even 4-hit stamcrit and removed the knockdown, confusion that comes with each hit.
Making them red-alert exclusive sounds like a bad joke

**Comment by Nebulacrity** [comment]
An emag should be able to bypass this restriction, in my opinion.

**Comment by nukularpower** [comment]
This is frankly idiotic.  Why do you keep trying to balance this unbalancable game by removing bits and pieces?   The overall effect is to reduce choice and railroad certain playstyles, though you think you are doing the opposite.  Its so gamey - we have stun prods now, in the present, but future space tech cant come up with the idea? 

Shortsighted and badly thought out nerf that should not have been immediately pushed through.    Buy hey, at least Xenobio are still gods, right

**Comment by LSommers** [comment]
The absolute state of coder/playerbase disconnect.

**Comment by SethLafuente** [comment]
brig physician RISE UP

**Comment by nukularpower** [comment]
"fuck you sec, no more nonlethals"   but hey, its okay, instead of blue alert all shift now we can be red alert all shift, right?

This is the gamiest, most meta solution possible

**Comment by SomeAngryMiner** [comment]
Holy cow. This is happening.

**Comment by OctusGit** [comment]
IM IN THE SCREENIE BOYS

**Comment by CCC23** [comment]
finally Stamina damage has been defeated time to kill everyone

**Comment by MCterra10** [comment]
put me in the montage

**Comment by MayumiBun** [comment]
aint no fucking way you keeping this holy shit. I will literally unironically support shitsec murderboning antags at this rate if this stays.

**Comment by ReturnToZender** [comment]
What the fuck is this. WHAT. HOW IS TAKING AWAY NON-LETHALS FROM SEC A SOLUTION.

**Comment by TheFakeElon** [comment]
esports ready codebase

**Comment by EyEy69** [comment]
Literally the best pr in years

**Comment by BobDanger-Coding** [comment]
But we still can make stunprods, right?

**Comment by SexmasterWasTaken** [comment]
Did you even consider buffing other weapons to make them comparable to stunbatons or do you people just hate fun

**Comment by Trubbl3** [comment]
the absolute state of neverplayer coders

**Comment by Jackal-boop** [comment]
cringe as hell

**Comment by ArthuroPep** [comment]
What the actual fuck

**Comment by ShamanSliph** [comment]
So sec gets less options for non-lehtally taking some one down. Unless the station is in a situation where that tool is generally not needed. Okay then.

**Comment by ReturnToZender** [comment]
While we're at it, let's remove security and just give everyone roundstart shotguns to shoot criminals with.

**Comment by Kapu1178** [comment]
nice bait.

**Comment by 567Turtle** [comment]
This is absolutely horrible, why don't you just make it so it takes longer to stun people with a stun baton? How I see it sec offs new best friend is going to be a baseball bat or a toolbox, there's no reason for sec to take people in non-lethally because its infinitely harder without the stun baton.

**Comment by khevy** [comment]
wild

**Comment by Jackal-boop** [comment]
@optimumtact

**Comment by LemonInTheDark** [comment]
Lead to issues on master so I've reverted it, we should try this idea more often, maybe in a year or three

BTW while I have you here, we've got some weird issues with byond rendering being yorked on older versions, update to latest 514 and things should be fixed. ❤️

**Comment by ApplePlastic** [comment]
This is horrid.

**Comment by mcmeiler** [comment]
Why don't we give sec tiny sized DNA locked lasers that shoot 30 burn damage projectiles while we're at it

**Comment by GoreDem** [comment]
im crying

**Comment by rellyreri** [comment]
its a nerf and a buff so they cancel out meaning this isn't a balance change

**Comment by Petnavis** [comment]
I can confirm no batons just lead to sec murdering everyone on sybil and lead to a lot of people tripping sec to take their shotguns and shooting it back at them and I can personally confirm it made it SIGNIFICANTLY easier to grief them with bolas because there's no reason threat besides disablers or back-up. Keep the change, I like making sec's life miserable

**Comment by optimumtact** [comment]
gaslightgatekeep


Sorry but I really couldn't resist when someone mentioned it in discord.

PS did anyone get to play a round with? Genuinely curious to see what sec would use instead lol

**Comment by Petnavis** [comment]
Yes, they used lethals because dodging disablers is extremely easy when the sec off you're messing with has a bolas restraining them. I think they also started putting them into crit with firelocks too. Sec doesn't realize how deadly their shotguns are and don't realize how dangerous it is to use them when someone steals said shotguns.

**Comment by optimumtact** [comment]
https://file.house/R4Oq.mp4

Meme by Somnus#7457

**Comment by DaydreamIQ** [comment]
We kind of got given a free pass to go full shitcurry during the round it was merged and admins gave sec all buckshot and shotguns. We should keep it that way imo that shit was hilarious

**Comment by Serxule** [comment]
> Did you even consider buffing other weapons to make them comparable to stunbatons or do you people just hate fun

i dont think making multiple weapons a 2 hit kill through all armor is a good idea

**Comment by Rustybuckets6601** [comment]
imho the stun meta is a good thing because it incentivizes non lethals to keep your victims alive. whether antag, sec, or just getting revenge on someone due to escalation. if stuns are the meta your first move is to immobolize them and cuff them, at that point you are able to kidnap them, execute them in a fashion beyond just hitting them with something over and over, or even just talk to them and release them. all of which encourage player interaction between the victim and the guy using the stuns. i think this change may accomplish the goal of having a bigger weapon variety, but decrease the amount of things you can realistically do with your victim. if i'm killed by some dude with a knife, that's cool, but if i'm kidnapped, cuffed to a chair, and my throat is slit in maint, that is way cooler.

**Comment by Weblure** [comment]
> PPS while I have reddits eyes: https://hackmd.io/97dtWNS5T2yfmyBcwwqyrw?view

Shitposting aside, I like the idea, assuming it ever gets coded and doesn't shit itself from Byond jank and latency issues. Though I feel like this would require yet another TG rebase to implement... What was the goal of the last one again?

**Comment by dendyd00m** [comment]
IM NOT OWNED IM NOT OWNED as i shrink into a corned cob

**Comment by Mothblocks** [comment]
it was a test to see if anyone would ping me about the changes and you all failed

**Comment by Iamgoofball** [comment]
everyone assumed you knew and approved since oranges was doing it

**Comment by Korusho** [comment]
v.v

**Comment by Mothblocks** [comment]
no i was watching cartoons

**Comment by Sealed101** [comment]
what a wonderful event to wake up to

**Comment by Jezzared** [comment]
> it was a test to see if anyone would ping me about the changes and you all failed

I was tempted to, but couldn't figure out a joke to go with it.

If I'm pinging a maintainer i gotta have a good bit to loosen the vibe.

**Comment by Mothblocks** [comment]
im not that threatening am i

**Comment by dendyd00m** [comment]
mothboob what cartoons were u watching i need to know for when i write the historical accounts of what happened here

**Comment by Mothblocks** [comment]
i was l;ying i was actually at work because he sent the message at 12 pm my time

**Comment by JohnFulpWillard** [comment]
we didn't tell you because we all wanted it to go through

**Comment by Booperkin1** [comment]
>Cloning removed
>Genetics nerfed into uselessness
>Shotguns removed now you have to use a shitty musket with fake buckshot LMAO
>Intents system that nobody had an issue with replaced with some confusing ass "combat mode"
>Stun batons that don't work
>LGBTQDJHFHHSJNHRI3II48UT+++ and Transgender pride pins
>Furry races and legalized ERP soon to come
Lmao this is what you get when you let a deranged terminally online idiot with a god complex and his cabal take full control of your code base and ignore all criticism. Enjoy your neutered remade for redditors walking corpe of a server, you retards deserve everything you get. Screenshot this post r/TransGender station will be dead within 5 years and no one will be playing it, this I promise you. Delete this post and ban me from the GitHub it won't change the truth.

**Comment by LemonInTheDark** [comment]
well done you are unable to read

**Comment by dendyd00m** [comment]
least insane vg player

**Comment by Booperkin1** [comment]
> least insane vg player

Least HRT overdosed TG player

**Comment by Booperkin1** [comment]
"This comment was marked as abuse."
Just like how you all gonna get marked off as another 41% statistic LMAO

**Comment by Jezzared** [comment]
Congratulations! You won the conversation!
[Claim your reward now!](https://m.youtube.com/watch?v=6n3pFFPSlW4)

**Comment by luzha-ss13** [comment]
sometimes i wonder if any of the /tg and somethingawful.com neckbeards from a decade ago had a slightest idea of what their favourite roleplaying game would turn into
Capture
inb4: "This comment was marked as misinformation."

**Comment by Niron3206** [comment]
_"Balance"_

File: code/game/objects/items/melee/baton.dm
Comment (timothymtorres) [reply code/game/objects/items/melee/baton.dm]: Also San balloon alerts are not supposed to be verbose since they disappear in a few seconds. Generally their text is short and sweet:  `"cant do that!"` `"need a wrench!"` etc.

**Comment by Terra1** [comment]
I'm sorry, but what the _fuck_ did I just read?

**Comment by Vect0r2** [comment]
Did you merge before you test merged?

**Comment by ZephyrTFA** [comment]
It was speed merged with maintainer consent, don't read the git history

**Comment by RuneQuester** [comment]
Gentlemen, the era of laser gun security has dawned

Make the criminals pay for their crimes in blood

**Comment by CRITAWAKETS** [comment]
smh should've been kept, would be funny to have an excuse to just body people as sec with absolutely fucked up weapons like the thermal pistol exploder

**Comment by VladinXXV** [comment]
> PS did anyone get to play a round with? Genuinely curious to see what sec would use instead lol

Whole station got shotguns, sec got buckshot. Not sure what ammo the crew was given if any. Apparently there was relaxed escalation but **I missed that part**, so I was going around using my disabler and pepperspray on threats that didn't warrant the buckshot. The first time I did this, another secoff immediately came in for the kill (not actually kill but dmg crit) after I already had them in stamcrit. For the terrible crime of stealing the lawyer's sunglasses. 

I was very confused.

Only other time was when we had a perma prisoner running around with a shotgun in brig. It was black orbit so the remaining threats I encountered warranted the buckshot (deswording traitor, nightmare. wizard was also around but we did not cross paths).

<details>
  <summary>Image</summary>

  image
</details>

<details>
  <summary>Video</summary>


https://user-images.githubusercontent.com/44104681/219900442-c4fe9061-7c06-40a2-bd89-3e4048f327ce.mp4
</details>

_Image and video courtesy of: donglesplonge#8541_

**Comment by JohnFulpWillard** [comment]
The solution to this is to keep this change permanently. Admins can't keep spawning buckshot forever

**Comment by MrrFiish** [comment]
My favorite part about the round this got merged was that I was AI, law 1 said harm could be done with shotguns, announced a tot put up a poster and watched them get buckshot by 5 officers.

**Comment by ZeWaka** [comment]
> Sorry but I really couldn't resist when someone mentioned it in discord.

:trollface:

**Comment by j-nengel** [comment]
No period at the end of the sentence telling you why you can't turn on your cobbled-together stunprod until the heads of staff declare red alert. 

This isn't just dumb, it's also lazy.

**Comment by optimumtact** [comment]
reading not this mans strongest suit

**Comment by MiniMeatwad** [comment]
I knew it, SS13 is just ancap science fantasy roleplay.

**Pull request merged**
